pkgs/development/python-modules/safe-pysha3/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptoolsCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "safe-pysha3";
+  version = "1.0.3";
+  format = "setuptools";
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Kd+lb9t5ney50BvdfbwPKAb0Ro0sKW+DtuN9hlMZF8I=";
+  };
+
+  nativeBuildInputs = [
+    setuptoolsCheckHook
+  ];
```

```suggestion
  nativeBuildInputs = [
    unittestCheckHook
  ];
```

The setupToolsCheckHook is unsupported as it run `python setup.py test` which is deprecated and going away.

---

It fails with errors when using `unittestCheckHook`. Should I disable the tests instead?

---

Looks like they're really using the setuptools test thingy. I guess explicitly going with `setuptoolsCheckHook` is better and doing it implicitly.

---

Running `tests.py` explicitly now in checkPhase.